<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectRecoveryWidget</class>
 <widget class="QWidget" name="ProjectRecoveryWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>194</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Project Recovery</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>It looks like Mantid has crashed recently. There is a recovery checkpoint available would you like to try it?</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item alignment="Qt::AlignVCenter">
    <widget class="QTableWidget" name="tableWidget">
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <column>
      <property name="text">
       <string>Checkpoint Time and Date</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Workspaces to recover</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="yesButton">
       <property name="text">
        <string>Yes</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="scriptWindowButton">
       <property name="text">
        <string>Just open in script window</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="startmantidButton">
       <property name="text">
        <string>Start mantid normally</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>yesButton</sender>
   <signal>clicked()</signal>
   <receiver>ProjectRecoveryWidget</receiver>
   <slot>onClickLastCheckpoint()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>11</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>scriptWindowButton</sender>
   <signal>clicked()</signal>
   <receiver>ProjectRecoveryWidget</receiver>
   <slot>onClickOpenLastInScriptWindow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>startmantidButton</sender>
   <signal>clicked()</signal>
   <receiver>ProjectRecoveryWidget</receiver>
   <slot>onClickStartMantidNormally()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>335</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>291</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onClickLastCheckpoint()</slot>
  <slot>onClickOpenLastInScriptWindow()</slot>
  <slot>onClickStartMantidNormally()</slot>
 </slots>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
